### ISSUE TITLE
uploader move method concat filepath twice

### ISSUE THREAD

**Issue by tnsezer** [magento/magento2#21812]
### Preconditions (*)

1. Magento  2.3
2. Magento catalog import/export module, Uploader file, move method

### Steps to reproduce (*)

1. I import a json data from external source and images come in external url like http://external-image.com/filename.jpg
2. I use import mapping and not use any 3. party method for it. 
3. When the module use Uploader move method, the method check the data, if the data is external, downloading it and writing in filepath. But if you see the image below, There are code issue because if the image is external, in the condition is the getRelativePath is used and concat $filePath and $fileName and to set $filePath again and next to the condition it happens again. It makes $filePath like /pat/folder/filename.jpgfilename.jpg

See lines 184. and 191.
[https://i.ibb.co/GQ8Hxk5/Screenshot-from-2019-03-18-09-43-56.png](code image)

### Expected result (*)

1. https://i.ibb.co/379Hkg1/Screenshot-from-2019-03-18-09-48-48.png

### Actual result (*)

1. https://i.ibb.co/P4cgbG0/Screenshot-from-2019-03-18-09-49-00.png

**Comment by magento-engcom-team** [comment]
Hi @tnsezer. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@tnsezer do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @kisroman. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by kisroman** [comment]
Hi @tnsezer. Can you pls provide your example of import file?

**Comment by tnsezer** [comment]
@kisroman  https://gist.github.com/tnsezer/d9801ed19b2b871de898f3c9bf80deb5

**Comment by kisroman** [comment]
This is duplicate of https://github.com/magento/magento2/issues/20098. This issue already fixed in branches 2.2-develop and 2.3-develop
